dataset-examples/bad-codex-data/submission/contributors.tsv

```diff
@@ -0,0 +1 @@
+orcid_id	name
```

Comment:
Having gone through doi issuance for the Brain Image Library project, the attached screen capture is how we collect the data for doi issuance by DataCite. We use the table to collect both creator and contributor information, having a field asking if the contributor is also a creator. A subset of the relevant DataCite tags for contributorType are also listed.  Contributors can be institutions or research groups which typiccally do not have orcid's but rather RORs or RRIDs.
<img width="900" alt="Contributors" src="https://user-images.githubusercontent.com/8113780/96751921-cb80b300-139b-11eb-89fa-722fc4d8c578.PNG">